components/explore-the-data-page/FilterContainer.js

```diff
@@ -65,7 +65,7 @@ const Fieldset = styled.fieldset`
   }
 
   legend {
-    padding: 0 0.5rem;
+    padding: 0 0.25rem;
```

Comment:
The "death location county" filter title was extending onto two lines, so I reduced the padding here to keep it on one line.